### PR TITLE
Healthcheck to all services

### PR DESCRIPTION
**Pull request by marianoleonardo**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

File: roles/report-manager/templates/report_manager_redis_deployment.yaml.j2 [hunk 1]
```diff
@@ -21,6 +21,14 @@ spec:
...
       - image: dojot/redis:6.0.4-alpine3.11
         name: report-manager-redis
+        readinessProbe:
+          exec:
+            command:
+              - redis-cli
+              - ping
+          failureThreshold: 30
+          periodSeconds: 1
+          timeoutSeconds: 3
```
Comment (gabrielifg) [inline roles/report-manager/templates/report_manager_redis_deployment.yaml.j2:31]: Leo, ele não vai ter o "livenessProbe"?

File: roles/rabbitmq/templates/rabbitmq_deployment.yaml.j2 [hunk 2]
```diff
@@ -20,8 +20,26 @@ spec:
     spec:
       restartPolicy: Always
       containers:
-      - image: rabbitmq:{{ dojot_rabbitmq_version }}
-        name: rabbitmq
+        - image: rabbitmq:{{ dojot_rabbitmq_version }}
```
Comment (danielapetito) [inline roles/rabbitmq/templates/rabbitmq_deployment.yaml.j2:23]: Nos outros serviços image está identado com containers..

File: roles/report-manager/templates/report_manager_deployment.yaml.j2 [hunk 3]
```diff
@@ -45,6 +45,12 @@ spec:
           value: "dojot-report-manager"
         - name: REPORTMANAGER_KEYCLOAK_URL
           value: "http://keycloak:8080"
+        readinessProbe:
+          exec:
+            command:
+              - curl
+              - -f
+              - http://localhost:9000/health
```
Comment (danielapetito) [inline roles/report-manager/templates/report_manager_deployment.yaml.j2:53]: readiness e liveness verifica a mesma URL ?

File: roles/report-manager/templates/report_manager_redis_deployment.yaml.j2 (hunk 1 above)
Comment (marianoleonardo) [reply roles/report-manager/templates/report_manager_redis_deployment.yaml.j2:31]: O livenessProbe já havia sido declarado para esse serviço. Está na linha 32.

Comment (marianoleonardo) [reply roles/report-manager/templates/report_manager_redis_deployment.yaml.j2:31]: image

File: roles/rabbitmq/templates/rabbitmq_deployment.yaml.j2 (hunk 2 above)
Comment (marianoleonardo) [reply roles/rabbitmq/templates/rabbitmq_deployment.yaml.j2]: Corrigido

File: roles/report-manager/templates/report_manager_deployment.yaml.j2 (hunk 3 above)
Comment (marianoleonardo) [reply roles/report-manager/templates/report_manager_deployment.yaml.j2]: Ajustei o readness probe para testar o endpoin `/ready`
Testei dentro do container e funcionou.

**Review by gabrielifg — APPROVED** [review]

**Review by danielapetito — APPROVED** [review]

**Pull request merged**